packages/dashboard/src/dashboard-actions/dashboardReducer.ts

```diff
@@ -192,18 +193,17 @@ export const dashboardReducer: Reducer<DashboardState, DashboardAction> = (
       };
 
     case 'UPDATE':
-      return updateDashboardState(state, {
-        ...action.payload.fieldsToUpdate,
+      return {
+        ...(state = { ...state, ...action.payload.fieldsToUpdate }),
```

Comment:
why have this `state = {...}` line?

Reply:
This must be an artifact of some previous commit, yes this shouldn't be here. will remove!